browser.py

```diff
@@ -1,12 +1,14 @@
+#!/usr/bin/env python
+
 import json
 import logging
 
-import gobject
-import webkit
+from gi.repository import WebKit
+from gi import _gobject as gobject
```

Comment:
@rhoit I'm used to see this import instead:

    from gi.repository import GObject

Is there any advantage on importing `_gobject`?